responsive-web-design/css-typography/index31.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <style>
+    html {
+      box-sizing: border-box;
+    }
+
+    *, *:before, *:after {
+      box-sizing: inherit;
+    }
+
+    body {
+      font-family: 'Open Sans', sans-serif;
+      font-size: 16px;
+    }
+
+    .label {
+      border: 2px solid black;
+      width: 270px;
+      margin: 20px auto;
+      padding: 0 7px;
+    }
+
+    h1 {
+      text-align: center;
+      margin: -4px 0;
+      letter-spacing: 0.15px;
+    }
+
+    p {
+      margin: 0;
+    }
+
+    .divider {
+      border-bottom: 1px solid #888989;
+      margin: 2px 0;
+    }
+
+    .bold {
+      font-weight: 800;
+    }
+
+    .right {
+      float: right;
+    }
+  </style>
+  <body>
+    <div class="label">
+      <header>
+        <h1 class="bold">Nutrition Facts</h1>
+        <div class="divider"></div>
+        <p>8 servings per container</p>
+        <p class="bold">Serving size <span class="right">2/3 cup (55g)</span></p>
+      </header>
+    </div>
+  </body>
+</html>
+
+<!-- Then change your h1 CSS rule to target "header h1". -->
```

Comment:
This is a new concept. Explain how this works (and how it does not target any other `h1` tags, only those within the header tag.